<launch>
	<node name="robot_action_server_node" pkg="robot_action" type="robot_action_server" output="screen"> 
		<param name="speed" type="double" value="0.1"/>
		<param name="angle_speed" type="double" value="0.1"/>
        <param name="marker_num" type="int" value="10"/>
        <param name="order_num" type="int" value="5"/>
		<!-- *3.14 -->
        <!-- MARKER -->
        <param name="Marker 1" type="string" value="(1,1)"/>
        <param name="Marker 2" type="string" value="(1,1)"/>
        <param name="Marker 3" type="string" value="(1,1)"/>
        <param name="Marker 4" type="string" value="(1,1)"/>
        <param name="Marker 5" type="string" value="(1,1)"/>
        <param name="Marker 6" type="string" value="(1,1)"/>
        <param name="Marker 7" type="string" value="(1,1)"/>
        <param name="Marker 8" type="string" value="(1,1)"/>
        <param name="Marker 9" type="string" value="(1,1)"/>
        <param name="Marker 10" type="string" value="(1,1)"/>
        <!-- ORDER_POS -->
        <param name="Order 1" type="string" value="(1,1)"/>
        <param name="Order 2" type="string" value="(1,1)"/>
        <param name="Order 3" type="string" value="(1,1)"/>
        <param name="Order 4" type="string" value="(1,1)"/>
        <param name="Order 5" type="string" value="(1,1)"/>
	</node>
</launch>